### ISSUE TITLE
solved build error on Windows with MSYS2

### ISSUE THREAD

**Issue by KarelLiebenberg** [openvenues/libpostal#514]
Hi!

I was checking out libpostal, and saw something that could be improved.

---
#### My country is
Ireland

---
#### Here's how I'm using libpostal
I'm chunking company addresses from BoardEx.com

---
#### Here's what I did
I ran pacman -S autoconf automake curl git make libtool gcc mingw-w64-x86_64-gcc as per your recommendation on Windows, using MSYS2

---
#### Here's what I got
Operation too slow. Less than 1 bytes/sec transferred the last 10 seconds
error: failed retrieving file 'msys2-runtime-devel-3.1.6-3-x86_64.pkg.tar.xz' from mirror.selfnet.de : expected download size exceeded

---
#### Here's what I was expecting
:: Proceed with installation? [Y/n] Y
:: Retrieving packages...
 msys2-runtime-devel-3.1.6-3-x86_64               5.4 MiB   127 KiB/s 00:44 [#########################################] 100%
 msys2-w32api-runtime-8.0.0.5683.629fd2b1-...  1834.7 KiB  5.06 KiB/s 06:02 [#########################################] 100%
 gcc-9.3.0-1-x86_64                              26.6 MiB   456 KiB/s 01:00 [#########################################] 100%

---
#### Here's what I think could be improved
I solved it by running pacman -S --disable-download-timeout autoconf automake curl git make libtool gcc mingw-w64-x86_64-gcc
May I recommend that you change the instruction to include --disable-download-timeout and warn users that it could take a few minutes to complete

**Comment by kyrill007** [comment]
Has the binary actually been built at the end?

**Comment by KarelLiebenberg** [comment]
No. On windows it did not build and on Ubuntu I could not get import to work from the pypostal binding. I ended up using the command line tool from the Ubuntu build. I piped in some input addresses, redirected the output to a text file. Tweaked the output to be valid json, and then used OpenRefine to turn that json into a spreadsheet.  Then I used the excel PROPER function to change out of all lower case.

**Comment by selva221724** [comment]
Hi All,

If you are trying to build libpostal in windows and facing many issues, pypostalwin is a package for you, feel free to open the issue if there is any

https://github.com/selva221724/pypostalwin

Please support the package to grow more and give start to this repo

If you don't want to use the python binding, download the 0rebuilt bundle that you can use from the pypostalwin repo
https://github.com/selva221724/pypostalwin#installation
